packages/admin/resources/views/pages/actions/select-action.blade.php

```diff
@@ -16,9 +16,15 @@
         @endif
 
         @foreach ($getOptions() as $value => $label)
-            <option value="{{ $value }}">
-                {{ $label }}
-            </option>
+            @if(($selected = $getSelected()) !== null)
+              @if($selected == $value)
```

Comment:
Maybe use @selected blade directive ?

Reply:
Is that supported in Laravel 8?

Reply:
Ops, nope, it was added 22 feb 2022, so L9.
Maybe for filament v3 then